test/CMakeLists.txt

```diff
@@ -88,9 +88,13 @@ if(CELERITAS_USE_Geant4)
   # Optional dependence on low-energy EM data and neutron data
   set(CELERITASTEST_G4ENV)
   foreach(_ds G4EMLOW G4ENSDFSTATE G4PARTICLEXS)
-    list(APPEND CELERITASTEST_G4ENV
-      "${Geant4_DATASET_${_ds}_ENVVAR}=${Geant4_DATASET_${_ds}_PATH}"
-    )
+    if(NOT DEFINED ENV{${Geant4_DATASET_${_ds}_ENVVAR}})
+      list(APPEND CELERITASTEST_G4ENV
+         "${Geant4_DATASET_${_ds}_ENVVAR}=${Geant4_DATASET_${_ds}_PATH}"
+       )
+    else()
+      message(STATUS "Value for ${Geant4_DATASET_${_ds}_ENVVAR} taken from the enviroment rather than the Geant4 cmake fragment")
```

Comment:
This prints
```
-- Value for G4LEDATA taken from the environment rather than the Geant4 cmake fragment
-- Value for G4ENSDFSTATEDATA taken from the environment rather than the Geant4 cmake fragment
-- Value for G4PARTICLEXSDATA taken from the environment rather than the Geant4 cmake fragment
```
at every (re)invocation of cmake.